iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/comparison_ops.mlir

```diff
@@ -0,0 +1,68 @@
+// Tests printing and parsing of comparison ops.
+
+// RUN: iree-opt -split-input-file -pass-pipeline='vm.module(iree-convert-vm-to-emitc)' %s | IreeFileCheck %s
+
+// CHECK: vm.module @module {
+vm.module @module {
+  // CHECK-LABEL: vm.func @cmp_eq_i32
+  vm.func @cmp_eq_i32(%arg0 : i32, %arg1 : i32) -> i32 {
+    // CHECK-NEXT: %0 = emitc.call "vm_cmp_eq_i32"(%arg0, %arg1) {args = [0 : index, 1 : index]} : (i32, i32) -> i32
+    %0 = vm.cmp.eq.i32 %arg0, %arg1 : i32
+    // CHECK-NEXT: vm.return
```

Comment:
```suggestion
```
Same as above. There is no real benefit as we only want to test the conversion.

Reply:
Fixed with latest push